src/backend/utils/page-builder.js

```diff
@@ -1,28 +1,142 @@
 /* eslint-disable object-curly-newline */
 /* eslint-disable indent */
 /* eslint-disable class-methods-use-this */
-
-const NotImplementedError = require('../utils/not-implemented-error')
 const Renderer = require('../../backend/utils/renderer')
 
 /**
- * PageBuilder class contains methods which allows you to create HTML content as JavaScript string
+ * PageBuilder class contains methods which allows you to create page content
+ * 
+ * @example
+ * 
+ * const { PageBuilder } = require('admin-bro')
+ * const ArticleModel = require('./../mongoose/article-model')
+ * const UserModel = require('./../mongoose/user-model')
+ * const ClientsModel = require('./../mongoose/clients-model')
+ * const CommentModel = require('./../mongoose/comment-model')
+ * const moment = require('moment')
+ * 
+ * class DashboardPage extends PageBuilder {
+ *   constructor(props) {
+ *     super(props)
+ *     this.title = 'My dashboard'
+ *   }
+ * 
+ *   async build() {
+ *     const articlesCount = await ArticleModel.countDocuments()
+ *     const publishedArticlesCount = await ArticleModel.find({ published: true }).countDocuments()
+ *     const unpublishedArticlesCount = articlesCount - publishedArticlesCount
+ *     const usersCount = await UserModel.countDocuments()
+ *     const comments = await CommentModel.find()
```

Comment:
lets add `limit` set to like 10, to the query. I know this is an example but it hits the eye (you cannot fetch all models from db beacuse it will kill memory)

Reply:
the same for ClientsModel